cobra.go

```diff
@@ -60,6 +61,10 @@ var EnableCommandSorting = defaultCommandSorting
 // EnableCaseInsensitive allows case-insensitive commands names. (case sensitive by default)
 var EnableCaseInsensitive = defaultCaseInsensitive
 
+// EnableTraverseRunHooks executes persistent pre-run and post-run hooks from all parents.
+// By default, only the first run hook to be found is executed.
```

Comment:
Can I suggest to change the second comment line to be:
// By default this is disabled, which means only the first run hook to be found is executed.

Reply:
Yes, sure.

I changed this and added some documentation in the force-push.